lcm-cmake/cpack.cmake

```diff
@@ -0,0 +1,44 @@
+# Package release version
+set(PACKAGE_RELEASE_VERSION 1)
+
+# Default CPack generators
+set(CPACK_GENERATOR "TGZ;STGZ")
+
+# Determine distribution
+execute_process(COMMAND lsb_release -si
+	OUTPUT_VARIABLE LINUX_DISTRO
+	OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Determine architecture
+execute_process(COMMAND uname -m
+	OUTPUT_VARIABLE	MACHINE_ARCH
+	OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Add CPack generator according to detected distribution
+if(LINUX_DISTRO STREQUAL "Debian" OR LINUX_DISTRO STREQUAL "Ubuntu" OR LINUX_DISTRO STREQUAL "LinuxMint")
+    set(CPACK_GENERATOR "${CPACK_GENERATOR};DEB")
```

Comment:
FYI, you can write `list(APPEND CPACK_GENERATOR DEB)` here (also `set(CPACK_GENERATOR TGZ STGZ)`).